src/routes/index.jsx

```diff
@@ -4,6 +4,7 @@ import { HashRouter, Routes as Switch, Route } from 'react-router-dom';
 import { NotFound } from 'Pages/NotFound';
 import { CreateVacancies } from 'Pages/CreateVacancies';
 import { EditVacancies } from 'Pages/EditVacancies';
+import { InterviewPlanning } from '../pages/InterviewPlanning';

```

Comment:
The alias. Instead of `../pages/InterviewPlanning` just `Pages/InterviewPlanning`

Reply:
Fixed!